Merged and resolved Pr/39

b-g opened the pull request:
# Conflicts:
#	.gitignore

b-g commented:
@Niels-NTG Thanks + merged!

I've tried to make it async (see below)  ... didn't really help ... so I simply scaled down the image by 50%. Now the performance is acceptable.

```
function async(func, callback) {
  setTimeout(function() {
    func();
    if (callback) {callback();}
  }, 0);
}
```

Niels-NTG commented:
Good. I think we should want to avoid using patterns like that. Teaching async JavaScript patterns is beyond the scope of the book, right?

b-g commented:
Yes it is beyond the scope of the "basic" P chapters ... but we always have the option to add more advanced things to the "gd" lib ... to basically provide the functionality but hide the complexity. BTW I've updated the gd lib yesterday and added Wacom Tablet support (not working chrome though).
